cloudinit/config/cc_scripts_vendor.py

```diff
@@ -3,12 +3,17 @@
 # Author: Ben Howard <ben.howard@canonical.com>
 #
 # This file is part of cloud-init. See LICENSE file for license information.
+"""Scripts Vendor: Run vendor scripts"""
 
-"""
-Scripts Vendor
---------------
-**Summary:** run vendor scripts
+import os
+
+from cloudinit import subp, util
+from cloudinit.config.schema import MetaSchema, get_meta_doc
+from cloudinit.distros import ALL_DISTROS
+from cloudinit.settings import PER_INSTANCE
 
+frequency = PER_INSTANCE
```

Comment:
Can drop this as it's unused and cloud-init defaults to PER_INSTANCE if not getattr(mod, "frequency").